### PR TITLE
Clean up themes

### LINKED ISSUE #934: Don't conflict both of class theme-dark and theme-light in new Debugger
In theme-dark,  class theme-light is available.
Don't conflict both of class theme-dark and theme-light in a new Debugger.

do-not-conflict-both-of-class-theme-dark-and theme-light-in-new-debugger

### PR DESCRIPTION
**Pull request by jasonLaster**
Fixes #934
Associated Issue: #370 

### Summary of Changes

* removes redundant class
* makes it easy to change themes locally

screen shot 2016-10-25 at 11 31 05 am
screen shot 2016-10-25 at 11 29 29 am

### CODE REVIEW

File: packages/devtools-local-toolbox/public/js/index.js [hunk 1]
```diff
@@ -47,6 +45,16 @@ function initApp() {
...
   const mount = document.querySelector("#mount");
 
+  // Using this static variable allows webpack to know at compile-time
+  // to avoid this require and not include it at all in the output.
+  if (process.env.TARGET !== "firefox-panel") {
+    const theme = getValue("theme");
+    theme == "dark" ? require("./lib/themes/dark-theme.css")
+                    : require("./lib/themes/light-theme.css");
+
+    document.body.parentNode.classList.add(`theme-${theme}`);
```
Comment (jasonLaster) [inline packages/devtools-local-toolbox/public/js/index.js]: The dark theme requires adding a class to the `<html>` element. This patch adds the class to both the html element and the root, which is just a way of being consistent in both cases

File: public/js/components/App.js [hunk 2]
```diff
@@ -58,8 +56,7 @@ const App = React.createClass({
 
   render: function() {
     return dom.div(
-      { className: classnames("debugger theme-body",
-                              { "theme-light": !isFirefoxPanel() }) },
+      { className: "debugger" },
```
Comment (jasonLaster) [inline public/js/components/App.js]: I pulled theme-body up to the Root element to be more consistent for other tools that use the toolbox, but I know we'll still want to figureout the toolbox / debugger api.

File: packages/devtools-local-toolbox/public/js/index.js [hunk 3]
```diff
@@ -47,6 +45,16 @@ function initApp() {
 function renderRoot(_React, _ReactDOM, component, _store) {
   const mount = document.querySelector("#mount");
 
+  // Using this static variable allows webpack to know at compile-time
+  // to avoid this require and not include it at all in the output.
+  if (process.env.TARGET !== "firefox-panel") {
+    const theme = getValue("theme");
+    theme == "dark" ? require("./lib/themes/dark-theme.css")
```
Comment (wldcordeiro) [inline packages/devtools-local-toolbox/public/js/index.js]: Is this doable?

```js
const theme = `./lib/themes/${getValue("theme")}-theme.css`
require(theme);
```

Comment (jasonLaster) [inline packages/devtools-local-toolbox/public/js/index.js]: nope. webpack doesn't like expressions and neither do I :)

File: packages/devtools-local-toolbox/public/js/index.js (hunk 1 above)
Comment (jlongster) [inline packages/devtools-local-toolbox/public/js/index.js]: Cool

**Comment by jlongster** [comment]
All of this makes sense, and doesn't look like a pile of hacks :) This makes us more consistent with the toolbox, which is nice. As long as you're 100% sure it doesn't bundle in the theme files for the toolbox (which it doesn't look like it should), this looks great.

**Pull request merged**